tests/promptflow/test_promptflow_model_export.py

```diff
@@ -0,0 +1,75 @@
+from pathlib import Path
+
+import pytest
+
+from promptflow import load_flow
+from promptflow._sdk.entities._flow import Flow
+
+import mlflow
+from mlflow import MlflowException
+
+
+def get_promptflow_example_model():
+
+    flow_path = Path(__file__).parent / "flow_with_additional_includes"
+    return load_flow(flow_path)
+
+
+def test_promptflow_log_and_load_model():
+    model = get_promptflow_example_model()
+    with mlflow.start_run():
+        logged_model = mlflow.promptflow.log_model(
+            model, "promptflow_model", input_example={"text": "Python Hello World!"}
+        )
+
+    loaded_model = mlflow.promptflow.load_model(logged_model.model_uri)
+
+    assert "promptflow" in logged_model.flavors
+    assert logged_model.signature is not None
+    assert str(logged_model.signature.inputs) == "['text': string]"
+    assert str(logged_model.signature.outputs) == "['output': string]"
+    assert isinstance(loaded_model, Flow)
```

Comment:
Could we test on loaded_model.predict(...) and compare to the original model's prediction result

Reply:
added